package.json

```diff
@@ -3,10 +3,16 @@
   "scripts": {
     "postinstall": "bower install",
     "build": "pulp build",
-    "test": "pulp test"
+    "test": "pulp test",
+    "spago-build": "spago build",
+    "spago-test": "spago -x ./test.dhall test"
   },
   "devDependencies": {
-    "pulp": "^12.3.0",
-    "purescript": "^0.12.0"
+    "pulp": "^15.0.0",
```

Comment:
Let's remove `pulp` from here and add replace it with `spago`.

Reply:
The thing is, if you do this, you can't publish

Reply:
Also,  of course, travis would fail.  Would you prefer to move to github actions?

Reply:
Ah ok. Maybe we keep `pulp` then in the deps. No need to move to github actions in this PR.

Reply:
I agree.  I'll just fix the big-integer dependency.  It means we can't complete this work until there's a new PS 0.14 release of purescript-bigints.